### ISSUE TITLE
[2.4-beta4 and 2.3] Breakout room assignment only working first time

### ISSUE THREAD

**Issue by PhMemmel** [bigbluebutton/bigbluebutton#13223]
**Describe the bug**
If a user doesn't accept the breakout room invitation he won't receive any new invitation for this room if assigned again.

**To Reproduce**
1. Mod opens breakout rooms for the first time, assigns user1 to room1
2. Modal appears as it's supposed to, user1 just closes modal
3. Mod: Reopening breakout room window and moving user1 to room2
4. Modal appears as it's supposed to, user1 just closes modal
5. From now whenever the moderator tries to reassign the user to a room he already just closed the invitation modal for the invitation modal doesn't pop up.
6. Shutting down alle breakout rooms and creating new ones make the modal again appear when moving user to a specific breakout room. See steps 1 to 5

BTW: Sometimes the invitation modal takes a really long time (up to 10s).

**Expected behavior**
Invitation should be shown again.

**Actual behavior**
No invitation showing up, user cannot be assigned to a room for which he declined the invitation at first.

**Desktop (please complete the following information):**
 - Probably not OS/browser related, tested with current Chrome/FF

**Additional context**
First reported in #13022, created separate issue as it's not certain they're identical

I can reproduce the same behavior on current test.bigbluebutton.org, HTML5 client build 1842

**Comment by PhMemmel** [comment]
Just tested most current `v2.4.x-release` after #13173 has been merged in my dev environment. This issue is still reproducible, so most likely this is different from #13022
